etc/profiles.yml

```diff
@@ -0,0 +1,7 @@
+microsoft-azure:
+  corosync_token: 30000
+  corosync_consensus: 36000
+  corosync_max_messages: 20
+  corosync_token_retransmits_before_loss_const: 10
+  sbd_watchdog_timeout: 60
+  sbd_msgwait: 120
```

Comment:
It probably makes sense to add a small comments in the beginning to describe those valid key definitions, example,  

# The valid profile names are: microsoft-azure, google-cloud-platform", "amazon-web-services.
#